src/main/kotlin/string/calculator/part/Calculator.kt

```diff
@@ -0,0 +1,20 @@
+package string.calculator.part
+
+import string.calculator.enum.Operator
+
+class Calculator {
+    fun calculate(numbers: MutableList<Long>, operator: MutableList<Operator>): Long {
```

Comment:
사소할 수 있지만 변수명이 단수냐 복수냐는 1개일지 복수개를 가지는지 한 눈에 파악할 수 있는 좋은 네이밍 습관이라고 생각해요. 😃

```suggestion
    fun calculate(numbers: MutableList<Long>, operators: MutableList<Operator>): Long {
```